pyuvm/s25_pyuvm_adapter.py

```diff
@@ -81,12 +84,12 @@ def get_parent_sequence(self):
 #    else:
 #        if(apb.op == READ):
 #            rw.kind   = UVM_READ
-#        elsif (apb.op == WRITE): 
+#        elsif (apb.op == WRITE):
 #            rw.kind   = UVM_WRITE;
 #        else:
 #            assert(0,"bus2reg -- Wrong operation type used for uvm_reg_bus_op")
 #        rw.addr      = apb.addr;
 #        rw.data      = apb.data;
 #        rw.status    = UVM_IS_OK;
-# 
-# ------------------------------------------------------------------------------
\ No newline at end of file
+#
+# ------------------------------------------------------------------------------
```

Comment:
fixing all the flake8 errors and using `uvm_not_implemeneted(self.header)` on `reg2bus` and `bus2reg`